docs/index.rst

```diff
@@ -23,6 +23,7 @@ Specific Doc Pages
    aicsimageio.readers.ome_tiff_reader.OmeTiffReader
    aicsimageio.readers.tiff_reader.TiffReader
    aicsimageio.writers.ome_tiff_writer.OmeTiffWriter
+   aicsimageio.writers.bfio_writers.OmeTiledTiffWriter
```

Comment:
I think remove this? This section of the docs is for the like "default writers and readers"